src/convnets/inception.jl

```diff
@@ -176,21 +178,303 @@ See also [`inception3`](#).
 
 !!! warning
     
-    `Inception3` does not currently support pretrained weights.
+    `Inceptionv3` does not currently support pretrained weights.
+"""
+struct Inceptionv3
+    layers::Any
+end
+
+function Inceptionv3(; pretrain = false, nclasses = 1000)
+    layers = inceptionv3(; nclasses = nclasses)
+    pretrain && loadpretrain!(layers, "Inceptionv3")
+    return Inceptionv3(layers)
+end
+
+@functor Inceptionv3
+
+(m::Inceptionv3)(x) = m.layers(x)
+
+backbone(m::Inceptionv3) = m.layers[1]
+classifier(m::Inceptionv3) = m.layers[2]
+
+@deprecate Inception3 Inceptionv3
+
+## Inceptionv4
+
+function mixed_3a()
```

Comment:
The other inception blocks for v3 allow the number of channels to be configured. It isn't really used right now, but maybe we can still support that here too?

Reply:
This follows the timm design - that too, allows the number of channels to be configure for Inception-v3 but not for v4 or ResNet-v2. My guess is that they're less configurable and so the numbers are too tricky to handle for edge case. But these blocks are never used independently anyways, so I think this should be fine - I'll try and see if they can be added if it still feels like something missing

Reply:
Okay let's skip the customization in this PR